### DISCUSSION TITLE
Having issue building .NET 8 Isolated durable function app with DurableFunctionsMonitor.DotNetBackend package

### DISCUSSION THREAD

**Discussion by ckimwilshire** [microsoft/DurableFunctionsMonitor#159] [Q&A]
Hello,

I tried installing the DurableFunctionsMonitor.DotNetBackend package in Visual Studio 2022 Professional and it looks like there is a dependency package called Microsoft.NET.Sdk.Functions that does not work with the isolated model. When I build the project, I get the following error:

"Microsoft.NET.Sdk.Functions package is meant to be used with in-proc function apps. Please remove the reference to this package in isolated function apps"

Does DurableFunctionsMonitor.DotNetBackend not work with the isolated model yet or is there any modification I can do? 

Thank you

**Comment by scale-tone** [comment]
@ckimwilshire , [the Isolated version of DfMon is here](https://github.com/microsoft/DurableFunctionsMonitor/tree/main/durablefunctionsmonitor.dotnetisolated#durablefunctionsmonitordotnetisolated), and [it is in preview](https://www.nuget.org/packages/DurableFunctionsMonitor.DotNetIsolated/).

Until then you have [plenty of other ways to use DfMon](https://github.com/microsoft/DurableFunctionsMonitor/wiki#overview) with your setup.

**Reply by ckimwilshire** [reply]
Thank you!
